src/fullpage/combine.py

```diff
@@ -12,6 +13,20 @@
 pattern = "\\node [below right,text width=1.86in] at (\LEFTMARGIN + %d * \CARDWIDTH, \TOPMARGIN + %d * \CARDHEIGHT) { \\fontsize{12}{0}\\selectfont %s };"
 postamble = "\\end{tikzpicture} }; \\end{tikzpicture} \\newpage \\backgroundreverse"
 
+def clean_content(line):
+    # Remove \REFERENCE{url} patterns
+    line = re.sub(r'\\REFERENCE\{.*\}', '', line)
+
+	# Remove \OPTIONAL{..} patterns
+    line = re.sub(r'\\OPTIONAL\{.*\}', '', line)
+
+    # Escape special LaTeX characters
+    # line = escape_latex(line)
+
+    # Trim any resulting double spaces
+    line = re.sub(r'\s+', ' ', line)
+    return line.strip()
```

Comment:
1. mind you, this entire thing is to workaround those commands in txt files you introduced. And tbh latex very much dislikes them.

Reply:
Oh?  I'd missed this.  There are two points to consider:

We've pretty esoteric shit here, so some benefits from references.

Some CAC decks often have pictures: https://www.amazon.de/-/en/Cards-Against-Humanity-Picture-Pack/dp/B098MVT2PB

Reply:
1. it's CAH
2. it's paid expansion. The original deck doesn't have pics: https://cdn.sanity.io/files/vc07edlh/production/024751665e12163130085836650f4f7387e2de0a.pdf
3. and anyway since we can't fork their repo we'll have to figure it ourselves

shouldn't be hard to add pics. But this code here is quite bad, so I'd start with refactoring.

Reply:
We cannot do multi-line cards cleanly using \\ or whatever?